src/main/java/net/dv8tion/jda/internal/entities/AbstractChannelImpl.java

```diff
@@ -166,8 +164,26 @@ public boolean isSynced()
     {
         if (getParent() == null)
             return true; // Channels without a parent category are always considered synced. Also the case for categories.
-        
-        return Helpers.deepEqualsUnordered(getParent().getPermissionOverrides(), getPermissionOverrides());
+
+        Stream<PermissionOverride> thisStream = getRolePermissionOverrides().stream().filter(
+                po -> po.getAllowedRaw() != 0 || po.getDeniedRaw() != 0
+        );
+        Stream<PermissionOverride> parentStream = getParent().getRolePermissionOverrides().stream().filter(
```

Comment:
What about members?

Reply:
didn't noticed, that I took the one for the roles, will change that